sdks/python/apache_beam/yaml/examples/README.md

```diff
@@ -229,6 +233,27 @@ gcloud dataflow yaml run $JOB_NAME \
   --region $REGION
 ```
 
+### Jinja
+
+Jinja [templatization](https://beam.apache.org/documentation/sdks/yaml/#jinja-templatization)
+can be used to build off of different contexts and/or with different
+configurations.
+
+Several examples will be created based on the already used word count example
+by leveraging Jinja templating engine for dynamic pipeline generation based on
+inputs from the user through `% include`, `% import`, and inheritance
+directives.
+
+% import:
+- [wordCount.yaml](#TODO: pending)
+
+% include:
+- [wordCount.yaml](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/yaml/examples/transforms/jinja/include/wordCount.yaml)
```

Comment:
Should this be `wordCountInclude.yaml`?

Reply:
It would also be good to give an example of invoking this (and how you would pass in appropriate parameters)

Reply:
I had it here before, but I moved it to the other Readme.  Similar pattern that Charles used on some of his ML work.

Reply:
> Similar pattern that Charles used on some of his ML work with Readme usage more in the applicable folder and not in the main one since multiple folders will be under Jinja.

I missed that Readme entirely when I did my review. SGTM

Reply:
Maybe we could just link to that folder or the Readme here then? I worry that it will be easy to miss if you just follow the link (basically how I reviewed this)